packages/eslint-plugin-orbit-components/src/rules/defaultTheme.ts

```diff
@@ -0,0 +1,125 @@
+import * as t from "@babel/types";
+import type { Rule } from "eslint";
+
+export const ERRORS = {
+  variableDeclaration: "Do not use defaultTheme as value, use useTheme() hook or ThemeConsumer",
```

Comment:
I think in the most cases you can consume it with the `ThemeConsumer`, but also by passing down the `theme` object, so it would be nice to specify that.